panoptes/routes/api.py

```diff
@@ -70,12 +70,14 @@ def get_job_of_workflow(workflow_id, job_id):
 
 @routes.route('/api/delete/<workflow_id>', methods=['GET'])
 def set_db_delete(workflow_id):
-    if(get_db_workflows_by_id(workflow_id)!=None):
+    if(get_db_workflows_by_id(workflow_id)==None):
```

Comment:
I think you need to check with `is` instead of `==` when you compare to None.

Reply:
done